src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -66,6 +65,7 @@ elseif (CLR_CMAKE_TARGET_ARCH_I386)
     add_definitions(-D_FILE_OFFSET_BITS=64)
 elseif (CLR_CMAKE_TARGET_ARCH_WASM)
     add_definitions(-DTARGET_32BIT=1)
+    add_definitions(-D_WASM_)
```

Comment:
We have stopped using the define names with underscores. Can this just use `TARGET_WASM` ?

Reply:
Fixed.